internal/handlers/sqlite/msg_insert.go

```diff
@@ -41,28 +40,31 @@ func (h *Handler) MsgInsert(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 	db := h.b.Database(params.DB)
 	defer db.Close()
 
+	iter := params.Docs.Iterator()
+	defer iter.Close()
+
 	res, err := db.Collection(params.Collection).Insert(ctx, &backends.InsertParams{
-		Ordered: params.Ordered,
-		Docs:    params.Docs,
+		Iter: iter,
 	})
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
 
 	replyDoc := must.NotFail(types.NewDocument(
-		"n", int32(res.InsertedCount),
+		"n", int32(res.Inserted),
 		"ok", float64(1),
 	))
 
-	if len(res.Errors) > 0 {
-		var errs *commonerrors.WriteErrors
-
-		for i := 0; i < len(res.Errors); i++ {
-			errs.Append(err, int32(i))
-		}
-
-		replyDoc = errs.Document()
-	}
+	// TODO
```

Comment:
I suppose this is okay like this for now?

Reply:
Added the issue link